config/config.go

```diff
@@ -75,7 +75,7 @@ func UpdateConfig() {
 func watchForConfigChange() {
 	defer func() {
 		if r := recover(); r != nil {
-			err := fmt.Errorf("failed while trying to update Config Variables with Error: %w", r.(error))
+			err := fmt.Errorf("cannot to update Config Variables: %v", r)
```

Comment:
new text doesn't appear to be syntactically correct
```suggestion
			err := fmt.Errorf("cannot update Config Variables: %v", r)
```